operators/GoogleCloudWeb3/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Google Cloud Web3",
   "website": "https://cloud.google.com/web3",
-  "description": "Google Cloud Web3 Team",
+  "description": "Build and scale faster with simple, secure tools and infrastructure for Web3. Our Web3 startup program gives you up to $200,000 USD over 2 years for Google Cloud and Firebase usage, access to our Discord channel, foundation grants, VIP event access, and more. Checkout https://cloud.google.com/web3.",
```

Comment:
could you remove `$` from this? I think our validation doesn't allow `$` - we will update that later but for right now it won't allow.